app/assets/javascripts/vanilla_nested.js

```diff
@@ -63,6 +63,7 @@
     if (wrapper.classList.contains('added-by-vanilla-nested')) {
       wrapper.remove();
     } else {
+      wrapper.classList.add('deleted-by-vanilla-nested');
```

Comment:
```suggestion
      wrapper.classList.add('removed-by-vanilla-nested');
```

To follow the word used in the other places.